tests/ipex/test_modeling.py

```diff
@@ -259,16 +257,13 @@ def test_ipex_patching_beam_search(self, test_name, model_arch, use_cache):
             GenerationConfig(max_new_tokens=4, num_beams=4, do_sample=True),
             GenerationConfig(max_new_tokens=4, num_beams=8, do_sample=True),
             GenerationConfig(max_new_tokens=4, num_beams=32, do_sample=True),
-            GenerationConfig(max_new_tokens=4, do_sample=not use_cache, top_p=1.0, top_k=5, penalty_alpha=0.6),
```

Comment:
The IPEXModel is not supported [_contrastive_search](https://github.com/jiqing-feng/transformers/blob/main/src/transformers/generation/utils.py#L1993) for now, we will try to enable it in the future.

Reply:
I see, could we then add a warning to state this is not supported (at least for transformers >= v4.39.0) and then upgrade it in the [setup.py](https://github.com/huggingface/optimum-intel/blob/v1.17.0/setup.py#L65) maybe ?